modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc

```diff
@@ -0,0 +1,280 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configure-rollout-route-traffic-using-openshift-routes_{context}"]
+= Configuring Argo Rollouts to route traffic by using OpenShift Routes
+
+You can use OpenShift Routes to configure Argo Rollouts to create routes, rollouts, and services.
+
+The following example procedure creates a route, a rollout, and two services. It then gradually routes an increasing percentage of traffic to a canary version of the application before that canary state is marked as successful and becomes the new stable version.
+
+.Prerequisites
+
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed the {gitops-title} on your {OCP} cluster.
+* You have installed Argo Rollouts on your {OCP} cluster. For more information, see xref:../argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc#gitops-creating-rolloutmanager-custom-resource_using-argo-rollouts-for-progressive-deployment-delivery[Creating a RolloutManager custom resource].
+* You have installed the {gitops-title} CLI on your system. For more information, see xref:../installing_gitops/installing-argocd-gitops-cli.adoc#installing-argocd-gitops-cli[Installing the {gitops-shortname} CLI].
+* You have installed the Argo Rollouts CLI on your system. For more information, see _Argo Rollouts CLI overview_.
+
+.Procedure 
+
+. Create a `Route` object.
+.. In the *Administrator* perspective of the web console, click *Networking* -> *Routes*.
+.. Click *Create Route*.
+.. On the *Create Route* page, click *YAML view* and add the following snippet:
++
+The following example creates a route called `rollouts-demo-route`:
```

Comment:
I think it might be a little awkward to have two sentences in a row ending with a colon like this. Here is one suggestion that can let you keep the content but shift around the formatting a bit:

```suggestion
.. On the *Create Route* page, click *YAML view* and add the following snippet.
The following example creates a route called `rollouts-demo-route`:
```

Reply:
Thanks, accepting the suggestion and applying your feedback.